latex/slides/07_arrays.tex

```diff
@@ -324,17 +325,14 @@ \subsection{}
 \end{frame}
 
 \begin{frame}[fragile]{Printing strings}
-	There is a placeholder for strings in printf: \textit{\%s}\\
-	It expects just the name of the array:
+	\begin{itemize}
+		\item You can print strings with printf
+		\item The placeholder for strings is \textit{\%s}\\
+		\item It expects an array of chars as an argument:\bigskip
```

Comment:
I liked the paragraph form more, because the items are quite short. Howver, I also see that the slide would be pretty empty otherwise.

Maybe add some more content? We could mention that character arrays can be passed to `puts` directly, just like string literals.